src/WebJobs.Script.WebHost/PreJIT/coldstart.jittrace

```diff
@@ -0,0 +1,2086 @@
+4714.7323-[System.Diagnostics.DiagnosticSource]System.Diagnostics.DiagnosticSourceEventSource+TransformSpec+PropertySpec+PropertyFetch+RefTypedFetchProperty`2<System.__Canon,System.__Canon>..ctor(Type,PropertyInfo)~System.Diagnostics.DiagnosticSourceEventSource+TransformSpec+PropertySpec+PropertyFetch+RefTypedFetchProperty`2[[System.__Canon,System.Private.CoreLib],[System.__Canon,System.Private.CoreLib]],System.Diagnostics.DiagnosticSource~2~0~.ctor
```

Comment:
I'd like to see the timestamps removed from the beginning of each line, and then the contents sorted. As written here, each time you update, you'll effectively replace this entire file, which is both big, and hard to analyze.

Reply:
Good point. it's updated now.